UnitTests/ProtectedMockFixture.cs

```diff
@@ -751,6 +771,16 @@ protected virtual string DoReturn(string a, string b)
 			protected virtual void SameFirstParameter(object a) { }
 
 			protected virtual void SameFirstParameter(object a, object b) { }
+
+			protected virtual FooBase OverloadWithDerived(MyBase myBase)
```

Comment:
Your totally right. Updated the method name accordingly. But a better name than "overloaded" also did not come into my mind. Finding good names is harder than coding ;)